spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/SparkExecutionPlanProcessor.scala

```diff
@@ -91,17 +90,13 @@ class SparkExecutionPlanProcessor(
       case r: WriteToDataSourceV2Exec =>
         r.writer match {
           case w: InternalRowMicroBatchWriter =>
-            try {
-              val streamWriter = w.getClass.getMethod("writer").invoke(w)
-              streamWriter match {
-                case sw: KafkaStreamWriter => KafkaHarvester.harvest(sw, r, qd)
-                case _ => Seq.empty
-              }
-            } catch {
-              case _: NoSuchMethodException =>
-                logDebug("Can not get KafkaStreamWriter, so can not create Kafka topic " +
-                  s"entities: ${qd.qe}")
-                Seq.empty
+            // Unfortunately, we cannot determine whether writer is kafka or not, before calling
```

Comment:
@dongjoon-hyun @bikassaha 
Unfortunately we can't get writer information even with reflection in Spark 2.3.x. It requires calling createInternalRowWriterFactory() to get it, which we normally expect trivial overhead, but not sure it applies all data sources.

Otherwise we can't get rid of custom patch on Spark 2.3.x. Is it OK to go?